src/main/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/domain/entities/ccd/Event.java

```diff
@@ -0,0 +1,63 @@
+package uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.entities.ccd;
+
+import com.fasterxml.jackson.annotation.JsonEnumDefaultValue;
+
+public enum Event {
+
+    START_APPEAL("startAppeal"),
+    EDIT_APPEAL("editAppeal"),
+    SUBMIT_APPEAL("submitAppeal"),
+    SEND_DIRECTION("sendDirection"),
```

Comment:
We don't need all these Events.  Maybe just keep 1 or 2 like `submitAppeal` and remove the rest

Reply:
Removed all other events except submitAppeal